### ISSUE TITLE
Add dashboard import (from grafana.net) by environment variable

### ISSUE THREAD

**Issue by xlson** [grafana/grafana#12895]
Copied from https://github.com/grafana/grafana-docker/issues/74

Original issue by @bvis:
One important feature in containers is the automation of initial configuration. In the grafana docker image this possibility is missing.

One container that runs this image is not useful without an initial manual configuration. This wouldn't be a problem if we don't count with the volatility of containers.

Adding the option of automatically import dashboards by environment variables would be a huge improvement.

For example:

```
docker run \
  --name grafana \
  -e GF_INSTALL_DASHBOARDS=123,456 \
  grafana/grafana
```
Which would import the dashboards available in grafana.net with these ids. If when they are imported the references to the datasources already exist in the container it would use them directly. If not, the first time, it should show the option of which datasources to use.

Depends on #10786

**Comment by shqear-mawdoo3** [comment]
I'd like to see this feature in Grafana :love_letter:

**Comment by shqear93** [comment]
I'd like to see this feature in Grafana  :love_letter:

**Comment by hugohaggmark** [comment]
Moving this to @grafana/backend-platform

**Comment by marefr** [comment]
/duplicate of #27059 

Referenced issue I would think is the way I think would make sense to support this.
